pkg/internal/checkup/trex/configfiles.go

```diff
@@ -0,0 +1,141 @@
+package trex
+
+import (
+	"fmt"
+
+	"github.com/kiagnose/kubevirt-dpdk-checkup/pkg/internal/config"
+)
+
+const (
+	CfgFileName                = "trex_cfg.yaml"
+	StreamPyFileName           = "testpmd.py"
+	StreamPeerParamsPyFileName = "testpmd_addr.py"
+)
+
+type Configs struct {
+	masterCPU                      string
+	latencyCPU                     string
+	trafficCPUs                    string
+	numOfTrafficCPUs               string
+	portBandwidthGB                string
+	trafficGeneratorEastMacAddress string
+	trafficGeneratorWestMacAddress string
+	DPDKEastMacAddress             string
+	DPDKWestMacAddress             string
+}
+
+func NewTrexConfigs(cfg config.Config) Configs {
```

Comment:
nit:

The above function is being called like this:
```go
trex.NewTrexConfigs()
```

The `Trex` in the function name is redundant IMO.

```suggestion
func NewConfigs(cfg config.Config) Configs {
```

This comment applies to the rest of the functions bellow as well.

Reply:
DONE